earthpy/plot.py

```diff
@@ -120,20 +120,25 @@ def plot_bands(
     try:
         arr.ndim
     except AttributeError:
-        "Input arr should be a numpy array"
+        raise AttributeError("Input arr should be a numpy array")
 
     if title:
-        if (arr.ndim == 2) and (len(title) > 1):
-            raise ValueError(
-                """Plot_bands() expects one title for a single
-                             band array. You have provided more than one
-                             title."""
-            )
-        elif not (len(title) == arr.shape[0]):
+        if isinstance(title, str):
+            title = [title]
+
+        # A 2-dim array should only be passed one title
+        if arr.ndim == 2 and len(title) > 1:
             raise ValueError(
-                """Plot_bands() expects the number of plot titles
-                             to equal the number of array raster layers."""
+                "plot_bands expects one title for a single"
```

Comment:
This will print as `...one title for a singleband array` - should it be `single band` instead of `singleband`?
```suggestion
                "plot_bands expects one title for a single "
```